camptix.php

```diff
@@ -2840,7 +2841,7 @@ function generate_attendee_report( $format ) {
 					'status' => ucfirst( $attendee->post_status ),
 					'txn_id' => get_post_meta( $attendee_id, 'tix_transaction_id', true ),
 					'coupon' => get_post_meta( $attendee_id, 'tix_coupon', true ),
-					'buyer_name' => empty( $buyer[0]->post_title ) ? '' : $buyer[0]->post_title,
+					'buyer_name' => empty( $buyer ) ? '' : $buyer,
```

Comment:
Would `$buyer ?? ''` work here? If so, it'd be a bit cleaner. I'm not sure off the top of my head, but I think it might have slightly different evaluation logic, though. I think it might be equivalent to `isset()` rather than `empty()`. That may not matter in this context, though.

Reply:
Removed the conditional, because `$buyer` will probably never be empty with current logic.